data/src/main/java/com/github/abdallahabdelfattah13/data/repositories/feed/InMemoryFeedRepoImpl.kt

```diff
@@ -41,17 +41,17 @@ class InMemoryFeedRepository : FeedRepository {
             description = "This is feed number: $feedId in your list"
         )
         feeds.add(feedCreated)
-        behaviorSubject.onNext(ArrayList<Feed>(feeds))
+        behaviorSubject.onNext(ArrayList(feeds))
         return Completable.complete()
     }
 
-    override fun getFeeds(): Observable<List<Feed>> = behaviorSubject
+    override fun getFeedsInMemory(): Observable<List<Feed>> = behaviorSubject
 
-    override fun getFeedArticles(feedId: Int): Flowable<List<Article>> {
+    override fun getFeedArticlesInMemory(feedId: Int): Flowable<List<Article>> {
         TODO("Not implemented yet")
```

Comment:
Is there any way to solve this TODO? If not, maybe we should remove this method.

Reply:
@AbdAllahAbdElFattah13 , what do yout thing about that?